rset.c

```diff
@@ -301,10 +306,26 @@ int main(int argc, char *argv[])
 					hl_range(host_labels[j]->name, HL_LABEL, 0, 0);
 					printf("\n");
 				}
-				if (tty_opt)
-					(void) ssh_command_tty(hostname, socket_path, host_labels[j], http_port);
-				else
-					(void) ssh_command_pipe(hostname, socket_path, host_labels[j], http_port);
+
+				int exit_code;
+				if (tty_opt) {
+					exit_code = ssh_command_tty(hostname, socket_path, host_labels[j], http_port);
+				} else {
+					exit_code = ssh_command_pipe(hostname, socket_path, host_labels[j], http_port);
```

Comment:
Let's leave off the braces for these single statements. (I may change my mind about this later, just a preference)